src/main/java/seedu/fma/storage/StorageManager.java

```diff
@@ -17,13 +17,13 @@
 public class StorageManager implements Storage {
 
     private static final Logger logger = LogsCenter.getLogger(StorageManager.class);
-    private AddressBookStorage addressBookStorage;
+    private LogBookStorage addressBookStorage;
```

Comment:
Remember to change addressBookStorage to logBookStorage (same for the lines below)